buildout.cfg

```diff
@@ -1,9 +1,9 @@
 [buildout]
 extends =
-    https://raw.github.com/collective/buildout.plonetest/master/test-4.3.x.cfg
+    https://raw.github.com/collective/buildout.plonetest/master/test-5.x.cfg
```

Comment:
@cewing I would prefer to let Travis do the job of testing this; we can change this later, when we achieve compatibility